### DISCUSSION TITLE
test: please +1 this announcement

### DISCUSSION THREAD

**Discussion by romange** [dragonflydb/dragonfly#84] [Announcements]
I am not sure how this discussion thingy works and whether its announcements reach a wide audience that starred the project.

if you saw this post, please plus one it :)

**Comment by AaronDewes** [comment]
![image](https://user-images.githubusercontent.com/67546953/171822664-af921fa4-bc97-41ca-b94f-be28b5dc8c73.png)

If it helps, this is how it appeared in my GitHub feed (starred the repo before).

**Reply by romange** [reply]
Yep, helps! Seems like working. And people reacting too :)
